libraries/AP_RCTelemetry/AP_CRSF_Telem.cpp

```diff
@@ -94,6 +94,13 @@ void AP_CRSF_Telem::setup_custom_telemetry()
         return;
     }
 
+    // check if passthru already assigned
+    if (AP::serialmanager().find_serial(AP_SerialManager::SerialProtocol_FrSky_SPort_Passthrough, 0) != nullptr) {
+       gcs().send_text(MAV_SEVERITY_CRITICAL,"CRSF: Passthru telem init fail, already assigned");
+       _custom_telem.init_done = true;
+       return;
+    }
+
```

Comment:
how about giving a hint on which port is conflicting?
```c++
    const int8_t frsky_port = AP::serialmanager().find_portnum(AP_SerialManager::SerialProtocol_FrSky_SPort_Passthrough,0); 
    if (frsky_port != -1) {
        gcs().send_text(MAV_SEVERITY_CRITICAL, "CRSF: passthrough telemetry conflict on SERIAL%d",frsky_port);
    }
```

Reply:
good idea! done...tested